cluster_kwargs.yaml

```diff
@@ -40,6 +40,12 @@ embarrassingly_parallel_cluster:
   backend_options:
     region: "us-east-1"  # Same region as dataset
 
+# For tests/workflows/test_uber_lyft.py
+uber_lyft_cluster:
+  n_workers: 30
```

Comment:
Is this one arbitrary? 
I have recommendations about the cluster size and instance type. In the tutorial we run this with 
```
    n_workers=15,
    worker_vm_types=["m6i.xlarge"],
``` 
but we are persisting the data, and casting objects to pyarrow strings, and other conversions to optimize memory.

Reply:
I based this on @mrocklin 's blog post draft:
https://docs.google.com/document/d/15VepxvOZoadTqM82cedxsyReq_hKZmMVZiKeFS_pADk/edit#
he's using 30 workers for that. I'll try your settings.